packages/rspack/prebundle.config.mjs

```diff
@@ -80,13 +80,7 @@ export default {
 		},
 		{
 			name: "@swc/types",
-			ignoreDts: true,
-			afterBundle(task) {
-				copyFileSync(
-					join(task.depPath, "index.d.ts"),
-					join(task.distPath, "index.d.ts")
-				);
-			}
+			ignoreDts: false
```

Comment:
We can remove `ignoreDts: false` as it is the default value

Reply:
done~